layouts/partials/header.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <html>
-    <head>
+    <head prefix="og: http://ogp.me/ns# fb: http://ogp.me/ns/fb# article: http://ogp.me/ns/article#">
```

Comment:
So what we need to do is only include the prefix when `opengraph` param is enabled, i think it is a good idea to include it, but since the `og` prefix is part of the initial context for RDF, it will work without it.